geoserver/webapp/src/docker/docker-entrypoint.d/00-geoserver-bootstrap

```diff
@@ -14,6 +14,6 @@ else
     sed -i 's:<serverURL>ldap\://localhost\:389</serverURL>:<serverURL>ldap\://ldap\:389</serverURL>:g' /mnt/geoserver_datadir/security/role/my_ldap/config.xml
     sed -i 's:<serverURL>ldap\://localhost\:389</serverURL>:<serverURL>ldap\://ldap\:389</serverURL>:g' /mnt/geoserver_datadir/security/usergroup/my_ldap/config.xml
 
-    echo 'Change rootLogger config according to docker setup'
-    sed -i 's:log4j.rootLogger=WARN, geoserverlogfile, stdout:log4j.rootLogger=WARN, stdout:g' /mnt/geoserver_datadir/logs/*
+    echo 'Change log config according to docker setup'
+    sed -i 's:stdOutLogging>false:stdOutLogging>true:g' /mnt/geoserver_datadir/logging.xml
```

Comment:
Then you also have to fix https://github.com/georchestra/geoserver_minimal_datadir/blob/master/logging.xml#L4